pkg/kubectl/explain/recursive_fields_printer.go

```diff
@@ -64,7 +65,12 @@ func (f *recursiveFieldsPrinter) VisitPrimitive(p *proto.Primitive) {
 
 // VisitReference is just a passthrough.
 func (f *recursiveFieldsPrinter) VisitReference(r proto.Reference) {
+	if value, ok := visitedReferences[r.Reference()]; ok && value {
+		return
+	}
+	visitedReferences[r.Reference()] = true
```

Comment:
you still don't have the `map[string]struct{}`

Reply:
If i change to map[string]struct{} i am getting **'pkg/kubectl/explain/recursive_fields_printer.go:33:25: type map[string]struct {} is not an expression'** error. So i thought you want to me make change from make(map[string]bool) to map[string]bool{} to initialize empty map.

Reply:
To initialize the empty map, you would have to do `map[string]struct{}{}`

Reply:
Thanks. I have changed to map[string]struct{}.